src/webview_provider/newDocsGenPanel.ts

```diff
@@ -62,32 +62,14 @@ export class NewDocsGenPanel
       queryManifestService,
       userService,
     );
-
```

Comment:
instead of sending separate message to webview, sending the tests data along with renderdocumentation message